lib/axon_onnx/deserialize.ex

```diff
@@ -658,6 +667,69 @@ defmodule AxonOnnx.Deserialize do
     {Map.put(axon, output_name, Axon.activation(inp, activation, name: output_name)), used_params}
   end
 
+  # Builds an Axon layer which returns a new layer with input values 
+  # concatenated on the given axis 
+  defp to_axon_concat(
+         %Node{attribute: attrs, input: inputs, output: [output_name]},
+         axon,
+         params,
+         used_params
+       ) when is_list(inputs) do
+    inputs = for inp <- inputs, do: input_or_param!(inp, params, axon, used_params)
+    %{ "axis" => axis } = options!(attrs)
+
+    {Map.put(axon, output_name, Axon.concatenate(inputs, [axis: axis, name: output_name])), used_params}
+  end
+
+  # Builds an Axon layer which returns a new layer upsampling the input
+  # layer. The scale activation layer must contain 1.0 as the first two values
+  # Each dimension value of the output layer is:
+  # output_dimension = floor(input_dimension * scale)
+  defp to_axon_upsample(
+         %Node{attribute: attrs, input: [inp, scale], output: [output_name]},
+         axon,
+         params,
+         used_params
+       ) do
+    inp = input_or_param!(inp, params, axon, used_params)
+    scale = input_or_param!(scale, params, axon, used_params)
+    updated_params = 
+      used_params
+      |> Map.put(output_name <> "_scale", scale)
```

Comment:
So the problem here is that `scales` will actually be updated during the training process - which is something we don't want. Looking at `Resize`, they mark `scales` as non-differentiable, which means they are constant. So we'll want to make these a constant part of the model as well. You can just remove this altogether then - just a note that I will need to update a few other layers to follow this convention as well.

```suggestion
```